src/Infrastructure/Twig/AppExtension.php

```diff
@@ -39,22 +40,27 @@ public function formatDateTime(\DateTimeInterface $date, ?\DateTimeInterface $ti
         return $dateTime->format($format);
     }
 
-    public function isFuture(\DateTimeInterface $date = null, \DateTimeInterface $time = null, \DateTimeInterface $reference = null): bool
+    public function isClientPastDay(\DateTimeInterface $date, \DateTimeInterface $today = null): bool
     {
-        if (!$date) {
-            $date = new \DateTimeImmutable('now');
+        if (!$today) {
+            $today = (new \DateTimeImmutable('now'))->setTimeZone($this->clientTimezone)->setTime(0, 0, 0, 0);
+        } else {
+            $today = \DateTimeImmutable::createFromInterface($today)->setTimeZone($this->clientTimezone)->setTime(0, 0, 0, 0);
```

Comment:
(Détail) On peut simplifier ce code : 1) obtenir un `DateTimeImmutable`, 2) convertir timezone et heure

```suggestion
        $today = $today ? \DateTimeImmutable::createFromInterface($today) : new \DateTimeImmutable('now');
        $today = $today->setTimeZone($this->clientTimezone)->setTime(0, 0, 0, 0);
```